content/machine-controller/guides/registry-authentication/_index.en.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Registry Authentication"
```

Comment:
This might also be part of OSM now, but should be double checked

Reply:
So what do? Remove the entire page?